docs/concepts/what-is-a-namespace.md

```diff
@@ -10,23 +10,54 @@ tags:
 
 A Namespace is a unit of isolation within the Temporal Platform.
 
-- [How to list Namespaces in a Cluster using tctl](/tctl/namespace/list)
-- [How to register a new Namespace using tctl](/tctl/namespace/register)
-- [How to set the Namespace for a Temporal Client](/application-development/foundations#set-namespace)
-- [How to view (describe) Namespace metadata and details using tctl](/tctl/namespace/describe)
-
 A single Namespace is still multi-tenant.
-You can use Namespaces to match the development lifecycle; for example, having separate `dev` and `prod` Namespaces.
-Or you could use them to ensure Workflow Executions between different teams never communicate; such as ensuring that the `teamA` Namespace never impacts the `teamB` Namespace.
 
+### Usage
+
+Namespaces are created on the Temporal Cluster, and provide a range of controls to achieve isolation on Workflow Executions.
+
+- You can use Namespaces to match the development lifecycle; for example, having separate `dev` and `prod` Namespaces.
 - If no other Namespace is specified, the Temporal Cluster uses the Namespace "default" for all Temporal SDKs and tctl.
-  - If you are deploying through Docker Compose or using the [auto-setup image](https://github.com/temporalio/docker-builds/blob/main/docker/auto-setup.sh) in a custom Docker Compose application, the Namespace "default" is created, through the auto-setup script.
-  - If you are deploying through the [Temporal Helm charts](https://github.com/temporalio/helm-charts), you can create the "default" Namespace by using tctl; for example, `tctl namespace default`.
-    We recommend using the default Namespace if you aren’t using multiple Namespaces.
+  See the [Registration](#registration) section for details.
 - **Case Insensitive**: Because of DNS, Namespaces are case insensitive on the network and routing side.
```

Comment:
This is not correct.

In OSS, namespace is case sensitive. You could create namespace foo and Foo, and they are 2 different namespaces.

In cloud, they are case insensitive. In fact, cloud only allow you to create lower case namespace names.

Reply:
updated this info